src/iOS/Xamarin.iOS/ViewControllers/SampleInfoViewController.cs

```diff
@@ -55,11 +63,14 @@ public override void ViewDidLoad()
 
         private void Initialize()
         {
+            CheckDarkMode();
```

Comment:
I recommend overriding UITraitCollectionDidChange to detect dark mode changes and reload the webviews as needed; currently if the theme changes while this VC is showing, the background changes but the text doesn't